packages/components/src/Footer/Footer.tsx

```diff
@@ -0,0 +1,194 @@
+import * as React from "react";
+
+import {
+  StyledFooter,
+  StyledFooterInner,
+  StyledFooterInnerLeft,
+  StyledFooterInnerRight,
+  StyledFooterLogo,
+  StyledFooterNav,
+  StyledSlogan,
+  StyledFooterSocial,
+  StyledFooterCopyright,
+} from "./FooterStyledComponents";
+
+export interface FooterProps {
+  currentYear: string;
+}
+
+export const Footer: React.SFC<FooterProps> = props => {
+  return (
+    <StyledFooter>
+      <StyledFooterInner>
+        <StyledFooterInnerLeft>
+          <StyledFooterLogo>
+            <a href="https://civil.co">
```

Comment:
don't we have a folder where icons live?

Reply:
Ah yeah, I just updated this PR with the proper icons (and added new ones where applicable).